content/_data/authors/chriskilding.adoc

```diff
@@ -0,0 +1,4 @@
+---
+name: "Chris Kilding"
+github: "chriskilding"
+---
```

Comment:
Up to you, but consider adding a short bio

Reply:
I'll stick with the 'man of mystery' approach for now